Singularity/translation.json

```
@@ -0,0 +1,131 @@
+﻿{
+	"tab-character": "Character",
+	"tab-characternotes": "Character Notes",
+	"tab-spaceship": "Spaceship",
+	"tab-spaceshipnotes": "Spaceship Notes",
+	"tab-scripts": "Scripts",
+	"title-character-u": "CHARACTER SHEET",
+	"title-characternotes-u": "CHARACTER NOTES",
+	"title-spaceship-u": "SPACESHIP SHEET",
+	"title-spaceshipnotes-u": "SPACESHIP NOTES",
+	"notes-u": "NOTES",
+	"modifiers-u": "MODIFIERS",
+	"modifiers-positive": "Positive",
+	"modifiers-negative": "Negative",
+	"attributes-u": "ATTRIBUTES",
+	"attributes-strength": "Strength",
+	"attributes-agility": "Agility",
+	"attributes-wits": "Wits",
+	"attributes-empathy": "Empathy",
+	"character-trauma-u": "TRAUMA",
+	"character-trauma-hitpoints-strength-plus-agility": "Hit Points (Str+Agi)",
+	"character-trauma-critical-injuries": "Critical Injuries:",
+	"character-trauma-mindpoints-wits-plus-empathy": "Mind points (Wits+Emp)",
+	"name": "Name:",
+	"character-background": "Background:",
+	"character-concept": "Concept:",
+	"character-icon": "Icon:",
+	"character-group-concept": "Group concept:",
+	"character-reputation": "Reputation:",
+	"character-personal-problem": "Personal Problem:",
+	"appearance-u": "Appearance",
+	"appearance-face": "Face:",
+	"appearance-clothing": "Clothing:",
+	"character-relationships-u": "RELATIONSHIPS",
+	"character-relationships-buddy-u": "BUDDY",
+	"radiation-u": "RADIATION",
+	"experience-u": "EXPERIENCE",
+	"character-skills-u": "SKILLS",
+	"character-skills-general-u": "GENERAL",
+	"character-skills-advanced-u": "ADVANCED",
+	"character-skills-dexterity-agility": "Dexterity (Agi)",
+	"character-skills-command-empathy": "Command (Emp)",
+	"character-skills-athletics-strength": "Athletics (Str)",
+	"character-skills-closecombat-strength": "Close Combat (Str)",
+	"character-skills-throw-strength": "Throw (Str)",
+	"character-skills-dodge-strength": "Dodge (Str)",
+	"character-skills-barter-empathy": "Barter (Edge)",.

```

```suggestion
	"character-skills-barter-empathy": "Barter (Edge)",
```
You had an extra dot here

---

Good catch thanks for the tip